Build failure: python3.13-duckdb-engine

### Nixpkgs version

- Unstable (26.05)

### Steps to reproduce

services.open-webui.enable = true;
failed for nixpkgs/master, nixpkgs-unstable, nixos-unstable

### Can Hydra reproduce this build failure?

Please select the Hydra Status.

### Link to Hydra build job

_No response_

### Relevant log output

```console
error: Cannot build '/nix/store/w28bvi452vxp4k0n9pxmbplqq8vc919v-python3.13-duckdb-engine-0.17.0.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/580i9i65hx5v772pz92c0xksgqvxzcxk-python3.13-duckdb-engine-0.17.0-dist
         /nix/store/wkk949i51g1z55dng7yb3pk7p4fa7xks-python3.13-duckdb-engine-0.17.0
       Last 25 log lines:
       > FROM pg_catalog.pg_type
       > WHERE pg_catalog.pg_type.oid = pg_catalog.pg_attribute.atttypid) != pg_catalog.pg_attribute.attcollation) THEN (SELECT pg_catalog.pg_collation.collname
       > FROM pg_catalog.pg_collation
       > WHERE pg_catalog.pg_collation.oid = pg_catalog.pg_attribute.attcollation) ELSE NULL END AS collation
       > FROM pg_catalog.pg_class LEFT OUTER JOIN pg_catalog.pg_attribute ON pg_catalog.pg_class.oid = pg_catalog.pg_attribute.attrelid AND pg_catalog.pg_attribute.attnum > $2 AND NOT pg_catalog.pg_attribute.attisdropped LEFT OUTER JOIN pg_catalog.pg_description ON pg_catalog.pg_description.objoid = pg_catalog.pg_attribute.attrelid AND pg_catalog.pg_description.objsubid = pg_catalog.pg_attribute.attnum JOIN pg_catalog.pg_namespace ON pg_catalog.pg_namespace.oid = pg_catalog.pg_class.relnamespace
       > WHERE pg_catalog.pg_class.relkind = ANY (ARRAY[$3, $4, $5]) AND pg_catalog.pg_table_is_visible(pg_catalog.pg_class.oid) AND pg_catalog.pg_namespace.nspname != $6 AND pg_catalog.pg_class.relname IN ($7, $8, $9, $10, $11) ORDER BY pg_catalog.pg_class.relname, pg_catalog.pg_attribute.attnum]
       > [parameters: (0, 0, 'r', 'p', 'f', 'pg_catalog', 'fake', 'IntervalModel', 'owner', 'table_with_binary', 'tb_notice')]
       > (Background on this error at: https://sqlalche.me/e/20/f405)
       > FAILED duckdb_engine/tests/test_pyarrow.py::test_fetch_arrow - sqlalchemy.exc.ProgrammingError: (_duckdb.CatalogException) Catalog Error: Table with name pg_collation does not exist!
       > Did you mean "pg_constraint"?
       >
       > LINE 6: FROM pg_catalog.pg_collation
       >              ^
       > [SQL: SELECT pg_catalog.pg_attribute.attname AS name, pg_catalog.format_type(pg_catalog.pg_attribute.atttypid, pg_catalog.pg_attribute.atttypmod) AS format_type, (SELECT pg_catalog.pg_get_expr(pg_catalog.pg_attrdef.adbin, pg_catalog.pg_attrdef.adrelid) AS pg_get_expr_1
       > FROM pg_catalog.pg_attrdef
       > WHERE pg_catalog.pg_attrdef.adrelid = pg_catalog.pg_attribute.attrelid AND pg_catalog.pg_attrdef.adnum = pg_catalog.pg_attribute.attnum AND pg_catalog.pg_attribute.atthasdef) AS "default", pg_catalog.pg_attribute.attnotnull AS not_null, pg_catalog.pg_class.relname AS table_name, pg_catalog.pg_description.description AS comment, NULL AS generated, NULL AS identity_options, CASE WHEN (pg_catalog.pg_attribute.attcollation != $1 AND (SELECT pg_catalog.pg_type.typcollation
       > FROM pg_catalog.pg_type
       > WHERE pg_catalog.pg_type.oid = pg_catalog.pg_attribute.atttypid) != pg_catalog.pg_attribute.attcollation) THEN (SELECT pg_catalog.pg_collation.collname
       > FROM pg_catalog.pg_collation
       > WHERE pg_catalog.pg_collation.oid = pg_catalog.pg_attribute.attcollation) ELSE NULL END AS collation
       > FROM pg_catalog.pg_class LEFT OUTER JOIN pg_catalog.pg_attribute ON pg_catalog.pg_class.oid = pg_catalog.pg_attribute.attrelid AND pg_catalog.pg_attribute.attnum > $2 AND NOT pg_catalog.pg_attribute.attisdropped LEFT OUTER JOIN pg_catalog.pg_description ON pg_catalog.pg_description.objoid = pg_catalog.pg_attribute.attrelid AND pg_catalog.pg_description.objsubid = pg_catalog.pg_attribute.attnum JOIN pg_catalog.pg_namespace ON pg_catalog.pg_namespace.oid = pg_catalog.pg_class.relnamespace
       > WHERE pg_catalog.pg_class.relkind = ANY (ARRAY[$3, $4, $5, $6, $7]) AND pg_catalog.pg_table_is_visible(pg_catalog.pg_class.oid) AND pg_catalog.pg_namespace.nspname != $8 AND pg_catalog.pg_class.relname IN ($9) ORDER BY pg_catalog.pg_class.relname, pg_catalog.pg_attribute.attnum]
       > [parameters: (0, 0, 'r', 'p', 'f', 'v', 'm', 'pg_catalog', 'tbl')]
       > (Background on this error at: https://sqlalche.me/e/20/f405)
       > ===== 7 failed, 29 passed, 35 skipped, 8 deselected, 102 warnings in 7.33s =====
       For full logs, run:
         nix log /nix/store/w28bvi452vxp4k0n9pxmbplqq8vc919v-python3.13-duckdb-engine-0.17.0.drv
```

### Additional context

_No response_

### System metadata

[spiage@i9:~/repos/nixos-config]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.63, NixOS, 25.11 (Xantusia), 25.11.20251229.89dbf01`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 - nixpkgs: `/nix/store/rv4z0vd7v0pkadinxd1c1xmd4b850ldr-source`

### Notify maintainers

@cpcloud 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/python-modules/duckdb-engine/default.nix
+++ b/pkgs/development/python-modules/duckdb-engine/default.nix
@@ -81,6 +81,14 @@ buildPythonPackage rec {
     "test_get_foreign_keys"
     "test_get_check_constraints"
     "test_get_unique_constraints"
+    # https://github.com/Mause/duckdb_engine/issues/1379
+    "test_reflect"
+    "test_get_multi_columns"
+    "test_table_reflect"
+    "test_comment_support"
+    "test_361"
+    "test_reflection"
+    "test_fetch_arrow"
   ];
 
   pythonImportsCheck = [ "duckdb_engine" ];

